static/app/gettingStartedDocs/electron/electron.tsx

```diff
@@ -148,7 +150,7 @@ const replayOnboarding: OnboardingConfig = {
               value: 'javascript',
               language: 'javascript',
               code: getReplaySDKSetupSnippet({
-                importStatement: `import * as Sentry from "@sentry/electron";`,
+                importStatement: `import * as Sentry from "@sentry/electron/renderer";`,
```

Comment:
had to look this up, but seems right: https://docs.sentry.io/platforms/javascript/guides/electron/#bundler-configuration

the replay docs also specify `@sentry/electron/renderer`